backend/tests/e2e/test_api_e2e.py

```
@@ -336,15 +336,15 @@
     # Note: Test settings are now handled by the isolated_e2e_settings fixture
 
     async def _validate_comprehensive_api_data(self, test_client, alert_id: str, session_id: str):
```

Ruff (0.12.2)

Unused method argument: `alert_id`

(ARG002)